src/metabase/actions/actions.clj

```diff
@@ -31,6 +32,11 @@
 
 (defmethod default-mapping :default [_ _])
 
+(defmethod default-mapping :table.row/common
+  [_ scope]
+  (when (= :table (:type scope))
```

Comment:
avoid nested mapping if it's used inside a row action.

remove this and `unified-execute-test` will fail.

Not really sure what's the correct way to use mapping here, this is what happens if we don't have this guard.
```clojure
;; mapping for built-in row action
{:configurable true,
 :dashcard-id 143,
 :inner-action {:action-kw :table.row/update,
                :mapping {:row :metabase-enterprise.data-editing.api/root,
                          :table-id 179}},
 :mapping {:row "::root", :table-id 179},
 :param-map {:name {:sourceType "row-data", :sourceValueTarget "name"}}}

;; inputs
{:row {:row {:id 3, :name "Braai tongs", :status "approved"}, :table-id 182},
 :table-id 182}

```

Reply:
I suspect the test here is what's wrong.

We should never need to do this.

Reply:
;; mapping for built-in row action
{:configurable true,
 :dashcard-id 143,
 :inner-action {:action-kw :table.row/update,
                :mapping {:row :metabase-enterprise.data-editing.api/root,
                          :table-id 179}},
;; there should be no mapping here, it should only be in `inner-action`
 :param-map {:name {:sourceType "row-data", :sourceValueTarget "name"}}}

;; inputs
{:row {:row {:id 3, :name "Braai tongs", :status "approved"}, :table-id 182},
 :table-id 182}

Reply:
One thing that confused me in your example is that the `table-id` is different between your mapping and input? What's supposed to be going on there? Is 182 for the table underlying the Editable containing the action?

Reply:
See `unified-execute-server-side-mapping-test`

Reply:
Ah! I see this was for the `built-in` action? That should be using `data-grid.row`.

Reply:
I guess there's no harm adding support for it as a primitive though.